client/js/components/map/publicdetail/Map.vue

```diff
@@ -1587,6 +1539,60 @@ export default {
       return layers
     },
 
+    getWMTSSource (layer) {
+      const layerArray = Array.isArray(layer.attributes.layers) ? layer.attributes.layers : layer.attributes.layers.split(',')
+      const url = this.addGetCapabilityParamToUrl(layer.attributes.url)
+
+      return $.ajax({
```

Comment:
Why did you change this back to ajax?

Reply:
because ajax has this tiny "async: false" (what ever that does in real - because technically its always async) Option, which is nor provided by axios. And the code is so messy, that I wasn't able to refactor it, that it works asynchronously.

Reply:
this is a bit mixced up. I initially wanted to cherrypick https://github.com/demos-europe/demosplan-core/pull/1866. I thought the cherrypick-PR would be faster that this one. Then I waited for the approval for 1866 and created this PR in the meanwhile which lead to the mess...